lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,28 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // If response.meta.pagination.links.next, this is from Rogue.
+      if (response.meta.pagination.links.next) {
+        var splitResponseUrl = response.meta.pagination.links.next.split("&");
```

Comment:
@DFurnes It actually seems like this happens in the way deeper in the files in [Fractal's Array Serialized](https://github.com/thephpleague/fractal/blob/master/src/Serializer/ArraySerializer.php#L112). I'm not sure how now how to create a custom file to extend this since it's way deeper than `IlluminatePaginatorAdaptor`?